lib/mappings/mappings.js

```diff
@@ -27,10 +27,15 @@ const amendMappingsBasedOnNyplSource = (data, nyplSource) => {
  */
 const mappingEntryToMarcQueryObject = (mapping) => {
   const marcQuery = Object.assign({}, mapping)
-  if (mapping.excludedSubfields) {
-    marcQuery.mappingOptions = { excludedSubfields: marcQuery.excludedSubfields }
-    delete marcQuery.excludedSubfields
-  }
+  // If either of these properties are set in the mappings json, copy them to the mappingOptions:
+  ; ['excludedSubfields', 'subfieldJoiner']
```

Comment:
for some reason i really love a leading semicolon... 👍🏻

Reply:
but on a more serious note - the following logic is a bit hard to parse because the variables have such similar names. can you summarize briefly in a comment?

Reply:
Added some inline comments to try to clarify